pages/CreateNewUserAccount/CreateNewUserAccount.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
```

Comment:
this file should be at `pages/users/create.tsx`. The other helper components can put them in the components folder instead